openshift/kustomize/services/content-historic/base/deploy.yaml

```diff
@@ -0,0 +1,221 @@
+              protocol: TCP
+          volumeMounts:
+            - name: ingest-storage
+              mountPath: /data
+          resources:
+            limits:
+              cpu: 150m
+              memory: 500Mi
+            requests:
+              cpu: 100m
+              memory: 200Mi
+          env:
+            # .NET Configuration
+            - name: ASPNETCORE_ENVIRONMENT
+              value: Staging
+            - name: ASPNETCORE_URLS
+              value: http://+:8080
+
+            - name: Logging__LogLevel__TNO
+              value: Information
+
+            # Common Service Configuration
+            - name: Service__ApiUrl
+              valueFrom:
+                configMapKeyRef:
+                  name: services
+                  key: API_HOST_URL
+            - name: Service__EmailTo__0
+              valueFrom:
+                configMapKeyRef:
+                  name: services
+                  key: EMAIL_FAILURE_TO
+
+            # Authentication Configuration
+            - name: Auth__Keycloak__Authority
+              valueFrom:
+                configMapKeyRef:
+                  name: services
+                  key: KEYCLOAK_AUTHORITY
+            - name: Auth__Keycloak__Audience
+              valueFrom:
+                configMapKeyRef:
+                  name: services
+                  key: KEYCLOAK_AUDIENCE
+            - name: Auth__Keycloak__Secret
+              valueFrom:
+                secretKeyRef:
+                  name: services
+                  key: KEYCLOAK_CLIENT_SECRET
+
+            - name: Kafka__Consumer__GroupId
+              valueFrom:
+                configMapKeyRef:
+                  name: content-tno-service
+                  key: KAFKA_CLIENT_ID
+            - name: Kafka__Consumer__BootstrapServers
+              valueFrom:
+                configMapKeyRef:
+                  name: services
+                  key: KAFKA_BOOTSTRAP_SERVERS
+
+            - name: Kafka__Producer__ClientId
+              valueFrom:
+                configMapKeyRef:
+                  name: content-tno-service
+                  key: KAFKA_CLIENT_ID
+
+            - name: Kafka__Producer__GroupId
+              valueFrom:
+                configMapKeyRef:
+                  name: content-tno-service
+                  key: KAFKA_CLIENT_ID
+            - name: Kafka__Producer__BootstrapServers
+              valueFrom:
+                configMapKeyRef:
+                  name: services
+                  key: KAFKA_BOOTSTRAP_SERVERS
+
+            # Content Service Configuration
+            - name: Service__MaxFailLimit
+              valueFrom:
+                configMapKeyRef:
+                  name: content-tno-service
+                  key: MAX_FAIL_LIMIT
+            - name: Service__UnpublishedIndex
+              valueFrom:
+                configMapKeyRef:
+                  name: content-service
+                  key: UNPUBLISHED_INDEX
+            - name: Service__PublishedIndex
```

Comment:
I do not believe the Content Server cares about a `PublishedIndex`.  Probably should fix the original source this file was copied from too.